safe/gui/tools/help/multi_buffer_help.py

```diff
@@ -0,0 +1,80 @@
+# coding=utf-8
+"""Help text for needs calculator."""
+
+from safe.utilities.i18n import tr
+from safe import messaging as m
+from safe.messaging import styles
+
+SUBSECTION_STYLE = styles.SUBSECTION_LEVEL_3_STYLE
+
+
+def multi_buffer_help():
+    """Help message for hazar layer generator dialog.
+
+    .. versionadded:: 4.0.0
+
+    :returns: A message object containing helpful information.
+    :rtype: messaging.message.Message
+    """
+
+    message = m.Message()
+    message.add(m.Brand())
+    message.add(heading())
+    message.add(content())
+    return message
+
+
+def heading():
+    """Helper method that returns just the header.
+
+    This method was added so that the text could be reused in the
+    other contexts.
+
+    .. versionadded:: 4.0.0
+
+    :returns: A heading object.
+    :rtype: safe.messaging.heading.Heading
+    """
+    message = m.Heading(tr('Multi buffer tool help'), **SUBSECTION_STYLE)
+    return message
+
+
+def content():
+    """Helper method that returns just the content.
+
+    This method was added so that the text could be reused in the
+    dock_help module.
+
+    .. versionadded:: 3.2.2
+
+    :returns: A message object without brand element.
+    :rtype: safe.messaging.message.Message
+    """
+    message = m.Message()
+    message.add(m.Paragraph(tr(
+        'This tool will generate polygon layer by multi-buffering input layer.'
+        ' To use this tool effectively:'
```

Comment:
It's better to have a space at the end of the previous line and not the beginning of this line.